src/content/docs/logs/logs-context/get-started-logs-context.mdx

```diff
@@ -161,14 +165,29 @@ For more on these options, see the agent-specific logs-in-context docs.
 
 ## Log metrics [#log-metrics]
 
-The logging metrics reported by <InlinePopover type="apm" /> agents are displayed on the <DoNotTranslate>**Logs**</DoNotTranslate> chart on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page. These metrics are separate from the [log forwarding feature](#apm-logs-defined) and cannot be disabled at the account level. To disable logging metrics, see the dedicated APM configuration docs (for example, [this `logging_metrics` config option for PHP](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#cfg-application_logging_metrics-enabled)).
+When you set up an <InlinePopover type="apm" /> agent, you automatically get a **Logs** chart of logging metrics on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page: 
+
+<img
+    style={{ align: 'left',maxWidth: '70%' }}
+    title="Screenshot showing the logging metrics chart"
+    alt="Screenshot showing the logging metrics chart"
+    src={logsLoggingMetrics}
+/>
+
+This chart shows a count of logs, and if you haven't disabled APM log forwarding, you can click on the chart links that will take you to the logs themselves. Even if you turn off APM log forwarding, this chart still shows the potential logs you could inspect if you had APM log forwarding enabled. 
 
 Logging metrics are reported via the `apm.service.logging.lines` attribute, as shown in the following query: 
 
 ```sql
 SELECT count(apm.service.logging.lines) FROM Metric WHERE (entity.guid = 'AN_ENTITY_GUID') LIMIT MAX SINCE 60 seconds AGO TIMESERIES
 ```
 
+If you don't want to see that logging metrics chart, you can turn it off&mdash;but not at the account level. To disable logging metrics, see the dedicated APM configuration docs for your agent (for example, [this `logging_metrics` config option for PHP](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#cfg-application_logging_metrics-enabled)). 
```

Comment:
"that chart" should be "the chart" I think

Should "APM configuration docs" be linking somewhere? It feels like it should even if logging metrics links somewhere as well

Reply:
@jeff-colucci I switched out "that" for "the." As far as adding a link to "APM configuration docs," these are spread out in different agents. So, we don't have a single page to send users to.